setup.py

```diff
@@ -56,19 +55,6 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 3',
     ],
-    
-    install_requires=[
-          'numpy',
-          'xarray',
-          'pandas',
-          'datetime',
-          'time',
-          'timeit',
-          'itertools',
-          'tqdm',
-          'subprocess',
-          'pathos.multiprocessing'
-      ],
```

Comment:
I replaced this by inserting a 'requirements.txt'. Also I am not sure what to do about pathos.multiprocessing which needs to be installed from conda forge :/

Reply:
I see, well, this is not equivalent. If you want to publish your package on pypi.org, for instance, the dependencies need to be  defined in `setup.py` (or `setup.cfg`). having a `requirements.txt` on your repository root is just `good practice`

But you can skip all the built-in libraries here, so better use

```python
    install_requires=[
          'numpy',
          'xarray',
          'pandas',
          'tqdm',
          'pathos'
      ],
```

(note that the name of the package is [pathos](https://pypi.org/project/pathos/), not `pathos.multiprocessing`).

> Also I am not sure what to do about pathos.multiprocessing which needs to be installed from conda forge

At this point, there is no possibility to cover this. But you should nevertheless list it in `install_requires`. It's the same for `numpy` and `pandas`, which should better be installed via conda.

As long as you do not yet provide a conda package for arc3o, I'd recommend to add a note like 

_We recommend to install the dependencies via `conda install -c conda-forge pandas tqdm pathos` as they might not work well using `pip`_.

to your installation instructions. When the user then installs your package via `pip install arc3o`, it will realize that `pathos`, etc. is installed already and won't download it from pypi.org.

Reply:
ok thanks, I'll do that then!